apps/challenges/challenge_config_utils.py

```diff
@@ -227,6 +227,7 @@ def get_value_from_field(data, base_location, field_name):
     "multiple_yaml_files": "There are {} challenge config YAML files instead of 1 in the zip file!",
     "yaml_file_read_error": "\n{} in line {}, column {}\n",
     "missing_challenge_title": "Please add the challenge title",
+    "invalid_challenge_title": "Challenge title contains invalid characters.",
```

Comment:
Error message is a bit vague, it should include the allowed characters range.

Reply:
@Suryansh5545 Please check now.